tap_google_analytics/client.py

```diff
@@ -253,6 +264,8 @@ def _query_api(self, report_definition, state_filter, pageToken=None) -> RunRepo
             metrics=report_definition["metrics"],
             date_ranges=[DateRange(start_date=state_filter, end_date=self.end_date)],
             limit=self.page_size,
+            metric_filter=report_definition["metric_filter"],
+            dimension_filter=report_definition["dimension_filter"],
```

Comment:
This is causing the tap to break if we don't define a metric filter or dimension in the `reports.json` definitions (or define them with a camelCase key, which would be reasonable given it's a JSON file).

That's because you define the keys with camelCase on lines 113-114, but tries to access them as snake_case here, which will lead to an invalid key error.